.github/workflows/reassurePerformanceTests.yml

```
@@ -0,0 +1,29 @@
+name: Reassure Performance Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: ['**']
+
+jobs:
+  perf-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run performance testing script
+        shell: bash
+        run: |
+          set -e 
+          BASELINE_BRANCH=${BASELINE_BRANCH:="main"}
+          git fetch origin
+          git switch "$BASELINE_BRANCH"
+          yarn install --force
+          yarn reassure --baseline
+          git switch --detach -
+          yarn install --force
+          yarn reassure --branch
+
+      - name: Run Danger.js
+        run: yarn danger ci
+        env:
+          GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
```

We should have an empty line at every file

```suggestion
          GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}

```